.github/workflows/release.yml

```
@@ -30,14 +30,12 @@ jobs:
         id: daily-version
       - if: steps.daily-version.outputs.created
         name: Create release
-        uses: actions/github-script@v6
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            await github.request(`POST /repos/${{ github.repository }}/releases`, {
-              tag_name: "${{ steps.daily-version.outputs.version }}",
-              generate_release_notes: true
-            });
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          tar -czvf "distribution.tar.gz" "./distribution/"
```

It'd be good to ensure that the generated zip does not contain the folder `distribution`. Probably 

```suggestion
          cd distribution
          zip -r refined-github.zip *
```

---

done